### PR TITLE
Added documentation for user profile management

### PR DESCRIPTION
**Pull request by WillTheDeveloper**
- Explained how to update user profile.
- Furthermore added how to clear a field if a user wished to,

### CODE REVIEW

**Review by JonruAlveus — APPROVED** [review]
Nice, thanks for the contribution! ❤️

File: docs/issues.md [hunk 1]
```diff
@@ -120,3 +120,17 @@ Label changes probably requires some explanation:
 
 If you're trying to populate the `Labels` collection by hand, you might hit
 some exceptional behaviour due to these rules.
+
+### Locking and unlocking
+
+The easiest way to lock and issue is to pass details of the issue then use ```Lock()``` to lock it.
+
+```csharp
+var issue = client.Issue.Lock(owner, repo, issueNumber);
```
Comment (JonruAlveus) [inline docs/issues.md:129]: I believe this has changed in a (very) recently merged PR. Could you take a look at the current docs to see if they need changing?

File: docs/user-management.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# User management
+
+With the GitHub API, you can manage your own user account.
+
+Theses details include the following:
```
Comment (JonruAlveus) [inline docs/user-management.md:5]: Theses > These

**Review by JonruAlveus — CHANGES_REQUESTED** [review]

File: docs/user-management.md (hunk 2 above)
Comment (WillTheDeveloper) [reply docs/user-management.md:5]: Well, that is a slightly embarrassing typo on my behalf.

File: docs/issues.md (hunk 1 above)
Comment (nickfloyd) [reply docs/issues.md:129]: It looks like this [PR](https://github.com/octokit/octokit.net/pull/2543) updated all of that.  The implementation is now:

`issuesClient.LockUnlock.Unlock()` and `issuesClient.LockUnlock.Lock()`

Comment (nickfloyd) [reply docs/issues.md:129]: @WillTheDeveloper I'd be glad to get this knocked out if you don't have the bandwidth to get them done - I'd need push access to your fork - let me know when you get the chance.  Thanks again for getting these changes made!

**Pull request closed without merging**